rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -50,13 +50,55 @@ $.fn.dataTable.ext.order['dom-checkbox'] = function  ( settings, col )
 }
 
 /*
- * This function is used to resort a table after its sorting datas were changed ( like sorting function below)
+ * This function is used to resort a table after its sorting data were changed ( like sorting function below)
  */
 function resortTable (tableId) {
   var table = $("#"+tableId).DataTable({"retrieve" : true});
   table.draw();
 }
 
+
+$.fn.dataTable.ext.search.push(
+    function(settings, data, dataIndex ) {
+
+        var param = window.location.hash;
+
+        if (param !== "" && window.location.pathname === "/rudder/secure/nodeManager/nodes") {
+
+            //! small parsing of the url argument to get the interval of compliance.
+            //! we return true if an error is triggered, so that everything goes normally
+            //! since it is not a critical error.
+
+            var from = param.indexOf('[');
```

Comment:
I don't think you should have [min,max] in your hash but a json that would be like this { complianceFilter { min : <value_min>, max : <value_max> } } and you should use JSON utilities to parse it